### PR TITLE
first pass on new backend service client

### PR DESCRIPTION
**Pull request by marenas**
first pass on new backend service client

### CODE REVIEW

File: RouteToDo/BackendRepository.h [hunk 1]
```diff
@@ -33,6 +33,7 @@
 #pragma mark - Actions
 
 - (void)toggleRouteFavoriteWithUser:(User *)user route:(Route *)route completion:(void (^)(NSError *error))completion;
+- (void)untoggleRouteFavoriteWithUser:(User *)user route:(Route *)route completion:(void (^)(NSError *error))completion;
```
Comment (yonpols) [inline RouteToDo/BackendRepository.h]: I don't think this is necessary. Toggle means its going to flip the value to true or false

**Comment by yonpols** [comment]
Can we remove parse from the pod file?

File: RouteToDo/BackendRepository.h (hunk 1 above)
Comment (marenas) [inline RouteToDo/BackendRepository.h]: makes sense. fixed.

**Comment by marenas** [comment]
will remove parse as well.

**Pull request merged**